install/kubernetes/helm/istio/values.yaml

```diff
@@ -219,6 +219,12 @@ global:
   # propagated, not recommended for tests.
   controlPlaneSecurityEnabled: false
 
+  # indicate the domain used in SPIFFE identity URL
+  # The default depends on the environment.
+  #   kubernetes: cluster.local
+  #   else:  default dns domain
+  trustDomain:
```

Comment:
```trustDomain``` is already added to this file, https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values.yaml#L344